features/dd-sdk-android-flags/src/main/kotlin/com/datadog/android/flags/internal/ExposureEventsProcessor.kt

```diff
@@ -69,13 +67,15 @@ internal class ExposureEventsProcessor(private val writer: RecordWriter) : Event
     private fun buildExposureEvent(flagName: String, context: EvaluationContext, data: PrecomputedFlag): ExposureEvent {
         val now = System.currentTimeMillis()
         return ExposureEvent(
-            timeStamp = now,
-            allocation = Identifier(data.allocationKey),
-            flag = Identifier(flagName),
-            variant = Identifier(data.variationKey),
-            subject = Subject(
+            timestamp = now,
+            allocation = ExposureEvent.Identifier(data.allocationKey),
+            flag = ExposureEvent.Identifier(flagName),
+            variant = ExposureEvent.Identifier(data.variationKey),
+            subject = ExposureEvent.Subject(
                 id = context.targetingKey,
-                attributes = context.attributes.mapValues { it.value }
+                attributes = ExposureEvent.Attributes(
+                    additionalProperties = context.attributes.mapValues { it.value }.toMutableMap()
```

Comment:
Is `mapValues { it.value }` call needed here? I think if type of `context.attributes` is `Map<String, String>` and type of `additionalProperties` is `MutableMap<String, Any?>`, doing `context.attributes.toMutableMap()` should be enough.